### ISSUE TITLE
About the Spring1944.net domain

### ISSUE THREAD

**Issue by frju365** [spring1944/spring1944.github.io#14]
Hello, 

I wanted to know if it's possible to have access to domain administration (record...) for future website/service of spring1944.
It's more and more difficult to add an adress, or subdomain to website. For example I wanted to add a subdomain for the media gallery on the website. And perhaps one day we will want to have a proper wiki or something like that. Having access to domain is quite essential for this and I don't think only one person even if it's an active member should have, alone, the access to the domain.

@tvo, are you the one who owns the domain ? Is it possible to give access to @jal76, @yuritch, @sanguinariojoe, @PepeAmpere, @specing and/or me (frju365) ? 

Thanks in advance for the answer and points of view,

frju365

**Comment by tvo** [comment]
Hi!

Indeed I own spring1944.eu, spring1944.net, spring1944.org. I checked my registrar though and I don't think there's a way to delegate access to any single domain.

That said, I'm happy to help out by adding a subdomain when needed.

-tvo

**Comment by frju365** [comment]
Hi,

mh... if it's possible to transfer it, for example on Gandi registrar, it could be nice (I could pay for it). In Gandi, we can create an orga and delegate ownership to a group as far as I remember.

Which registrar is it ?

Well, I try to find a long-time solution, but I don't surely have the best.

For the subdomain, media.spring1944.net should be fine. For the A record value (ip), it's @specing who wanted to host the screenshots gallery, so I don't know. 

Another question : why do we have 3 domain, whereas only one is used ? Perhaps we could, either share domain (see if we really need 3, and if we really needed it, perhaps having several owner who could renew it), either saving money in deleting one or two of the three domains we have (don't know how many it costs).


Thanks anyway,

frju365

**Comment by frju365** [comment]
For the Gandi feature, I found it : https://docs.gandi.net/en/sharing_products/teams/index.html Don't know, if it's a solution or not.

**Comment by tvo** [comment]
Let me read up on that a little over the weekend. I'm not opposed to transfer but want to make sure it goes to the right people.

As for why 3 domains, I've just been lazy. I bought all 3 back in the day and have been too lazy to figure out which one is really used, and the money for 3 domains hasn't been a problem for me.

Anyway, for the short term please let me know any subrecord you'd want created, including type and IP, and I can set it up.

**Comment by PepeAmpere** [comment]
Hi, I'm fine with passing some domene rights on people @sanguinariojoe, @jal76, @yuritch, @specing but I would be careful in delegation of some powers on @frju365 exclusively. He has historical record of mental instability. On the other hand I do not want to prevent him to contribute to community again once he says he want to help. 

If anyone @sanguinariojoe, @jal76, @yuritch, @specing would be new owners/maintainers of the domene with full rights and @frju365 gets FTP or other kind of access, I'm ok with that.

**Comment by frju365** [comment]
I'm still demanding me, @PepeAmpere, why I help ppl like you. :-/  As I said, I don't want rights on it. I explained it. As I said, I'm not mentally unstable, I was just angry at one moment and what you said at that time didn't help anybody to solve the situation.
 
Moreover, it was about gitlab rights, and you never want to go on gitlab, so I don't understand what you did have to do with this situation. I was the one who took the initiative to create a Gitlab organization. Anyway, you are very opposed to anything I did and I do.

What you say right now is not helping a dying community, and quite offense ppl who try to help it (at least me). I'm 21 and I'm community manager, dev and more in much more communities. I'm only helping Spring1944, because some ppl had worked hardly to make an awesome game and I love it. 

I apologized, but you keep harassing me with that. I tried to have an explanation with you in private to know what you want, you never answered. I'm really fed up. 

However, I don't want any rights, I just want  somebody close to project to have access.
